givenergy_modbus/model/inverter.py

```diff
@@ -241,8 +238,46 @@ class InverterRegisterGetter(RegisterGetter):
         #
         # Holding Registers, block 180-239
         #
+
         "enable_standard_self_consumption_logic": Def(C.bool, None, HR(199)),
         "cmd_bms_flash_update": Def(C.bool, None, HR(200)),
+        "charge_target_soc_1": Def(C.uint16, None, HR(242)),
+        "charge_slot_2": Def(C.timeslot, None, HR(243), HR(244)),
```

Comment:
This seems to be a duplicate "charge_slot_2" - it's also at HR(31)/HR(32) ???

(reported by pylint)